src/DocumentDB.ChangeFeedProcessor/PartitionManagement/IPartitionSynchronizer.cs

```diff
@@ -14,6 +14,6 @@ internal interface IPartitionSynchronizer
     {
         Task CreateMissingLeasesAsync();
 
-        Task<IEnumerable<ILease>> SplitPartitionAsync(ILease lease);
+        Task<IEnumerable<ILease>> SplitPartitionAsync(ILease lease, string lastContinuationToken);
```

Comment:
good question -> honestly, I dont know. I didnt want to manipulate old lease but pass the last known token from the exception because that Lease object is not touched at all while processing.